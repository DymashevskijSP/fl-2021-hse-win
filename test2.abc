Sigma:"a,b,c"
Q:"q_0,q_1,q_2,q_3" 
q0:"q_0"
Terminals:"q_3"
delta: '(q_0, a)->q_1, (q_0, b)->q_0, (q_0, c)->q_0, (q_1, a)->q_1, (q_1, b)->q_2, (q_1, c)->q_0, (q_2, a)->q_1, (q_2, b)->q_0, (q_2, c)->q_3, (q_3, a)->q_3, (q_3, b)->q_3, (q_3, c)->q_3'
