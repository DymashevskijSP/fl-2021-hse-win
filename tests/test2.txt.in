A\,='B','C'/*the best grammar*/
/*/* some comment  */*/
