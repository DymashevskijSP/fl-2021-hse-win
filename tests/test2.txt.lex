LexToken(RULENAME,'A,',1,0)
LexToken(EQ,'=',1,3)
LexToken(PLAINTEXT,'B',1,4)
LexToken(COMMA,',',1,7)
LexToken(PLAINTEXT,'C',1,8)
LexToken(MULTILINE_COMMENT,'/*the best grammar*/',1,11)
LexToken(MULTILINE_COMMENT,'/*/* some comment  */*/',2,32)
