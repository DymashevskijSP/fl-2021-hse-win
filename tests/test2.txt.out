Node:     A, =  B , C /*the best grammar*/ /*/* some comment  */*/: Children    A, =  B , C /*the best grammar*/ | /*/* some comment  */*/ |
Node:    A, =  B , C /*the best grammar*/: Children   A, =  B , C | /*the best grammar*/ |
Node: /*/* some comment  */*/: Children EMPTY
Node:   A, =  B , C: Children  A, =  B , C |
Node: /*the best grammar*/: Children EMPTY
Node:  A, =  B , C: Children A, | = |  B , C |
Node: A,: Children EMPTY
Node: =: Children EMPTY
Node:  B , C: Children B | , | C |
Node: B: Children EMPTY
Node: ,: Children EMPTY
Node: C: Children EMPTY